packages/client/src/index.tsx

```diff
@@ -7,15 +7,31 @@ import { UserProvider } from 'components/UserContext';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { GlobalStyle } from 'GlobalStyle';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient({
```

Comment:
Is this here just to wrap all errors?

Reply:
in future we can put other config right now we don't need much since we don't use rest a lot

Reply:
But I mean, is `QueryClientProvider` necessary? Or did you put it just for catching the errors?

Reply:
it is yes, you need it to initialize the cache, then you can add options, those are not mandatory but I added those for now